docs/source/examples/vision/guides/formatters_validating.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>his is because although our labels are in the right structure but the content inside is not valid. -&gt; his is because although our labels are in the right structure, the content inside is not valid. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>